account_fx_spot/views/account_fx_view.xml

```diff
@@ -80,6 +79,17 @@
                         </group>
                         <group name="rate" string="Spot Exchange Rate">
                             <field name="rate" nolabel="1"/>
+                            <div>
+                                <field name="in_currency_id" readonly="1"/>
+                                /
+                                <field name="out_currency_id" readonly="1"/>
+                            </div>
```

Comment:
The issue are these divs where I use again in_currency_id and out_currency_id.

The desired effect is this:
![image](https://user-images.githubusercontent.com/23449160/38079839-2bb385c0-3340-11e8-8e94-185cfd1b0135.png)

What other options do I have? compute a string field to display it? any better idea?

Reply:
@lreficent Since rates are just for information i suggest to change those one to text and compute rate + in_currency + out_currency (and inverse), otherwise you have to add another fields for getting the currency names, so in total 4 fields, and to adjust tests after not to check float value of rate.

Reply:
just typed "outgoing" and "incoming", I think that works for now. I also switched the position of the incoming and outgoing groups as you suggested.